### ISSUE TITLE
.rubocop_todo incorrectly documents Style/DoubleNegation autocorrection safety

### ISSUE THREAD

**Issue by JoeCohen** [rubocop/rubocop#10704]
`.rubocop_todo .yml` incorrectly says that Style/DoubleNegation is **safely** autocorrectable

--------

## Expected behavior
.rubocop_todo.yaml should say
```yaml
# This cop supports unsafe autocorrection (--autocorrect-all).
Style/DoubleNegation:
```

## Actual behavior
.rubocop_todo.yaml says:
```yaml
# This cop supports safe autocorrection (--autocorrect).
Style/DoubleNegation:
```

## Steps to reproduce the problem

1. Add an offense, e.g.
```ruby
!!something
```
2. Regenerate the autoconfig file
```sh
rubocop --auto-gen-config
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler.
If you see extension cop versions (e.g. `rubocop-performance`, `rubocop-rspec`, and others)
output by `rubocop -V`, include them as well. Here's an example:

```
$ rubocop -V
1.30.0 (using Parser 3.1.2.0, rubocop-ast 1.18.0, running on ruby 2.7.6 x86_64-linux)
  - rubocop-performance 1.14.0
  - rubocop-rails 2.14.2

```

**Comment by ydah** [comment]
Can I ask you to try to upgrade the version to RuboCop 1.30.1? Perhaps that problem has been solved by https://github.com/rubocop/rubocop/pull/10673 😀

**Comment by JoeCohen** [comment]
Thank you!
That indeed solves the problems of the false "supports safe autocorrection" in `.rubocop_todo.yaml`.
I will close this Issue
